src/status_im2/constants.cljs

```diff
@@ -372,7 +372,9 @@
 ;; wallet
 (def ^:const mainnet-chain-id 1)
 (def ^:const optimism-chain-id 10)
+(def ^:const optimism-test-chain-id 420)
 (def ^:const arbitrum-chain-id 42161)
+(def ^:const arbitrum-test-chain-id 421613)
 (def ^:const goerli-chain-id 5)
```

Comment:
Should we name to mainnet test id?

Reply:
@J-Son89 this is test id of arbitrum test net

Reply:
I'm taking about id 5 btw

Reply:
i.e so we have 
```
; prod
(def ^:const mainnet-chain-id 1)
(def ^:const optimism-chain-id 10)
(def ^:const arbitrum-chain-id 42161)

; test
(def ^:const mainnet-chain-id 5)
(def ^:const optimism-test-chain-id 420)
(def ^:const arbitrum-test-chain-id 421613)
```

Reply:
@J-Son89 Idk should we do that? 🤔  It is called `Goerli` https://chainlist.org/?search=goerli&testnets=true

Reply:
Let's leave it as is 👌